device/send.go

```diff
@@ -163,7 +167,8 @@ func (peer *Peer) SendHandshakeResponse() error {
 	peer.timersAnyAuthenticatedPacketTraversal()
 	peer.timersAnyAuthenticatedPacketSent()
 
-	err = peer.SendBuffer(packet)
+	// TODO(raggi): allocation could be avoided
```

Comment:
This TODO(raggi) thing still here. Mentioned this earlier. Get rid of all of these.

Reply:
Removed in https://github.com/WireGuard/wireguard-go/pull/64/commits/23c2269626d980c95958a2b0fc166601a4a6c4dd.